lesson07/assignment/spaceland/src/App.vue

```diff
@@ -0,0 +1,64 @@
+<!--this is where to place the main nav for the site, as it will be repeated on each page-->
+
+<template>
+  <div id="app">
+    <head>
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+      <link rel="stylesheet" href="css/style.css" />
```

Comment:
This is creating an error in the web console since it does not exist. You can remove this line

Reply:
Thank you! I fixed it.